stable/xray/values.yaml

```diff
@@ -280,8 +280,8 @@ analysis:
   service:
     type: ClusterIP
   ## Container storage limit
-  storage:
-    sizeLimit: 10Gi
+  persistence:
+    size: 10Gi
```

Comment:
I think 10GB is less for server. On the test environment, can we do a full DB sync and check the size? I think this data is eventually cleaned up but we just want to make sure that the default values are good.